### ISSUE TITLE
pydoc-user-modules: Wrong type argument: sequencep, Traceback

### ISSUE THREAD

**Issue by JosefMikkelsen** [statmobile/pydoc#12]
Versions:
- GNU Emacs 24.5.1 (i586-pc-linux-gnu, GTK+ Version 3.18.9) of 2016-04-08 on x86-csail-01, modified by Debian
- pydoc 20160403.1733 from Melpa, installed from list-packages

When I run M-x pydoc and hit enter, I immediately get this error message:

`pydoc-user-modules: Wrong type argument: sequencep, Traceback`

I am not prompted for the python module/function name.

I have determined that this happened because I did not have pip installed globally, I tend to use pip exclusively in virtualenvs.

Is it implicit that one's emacs must be virtualenv-aware to use this? Or would it be a good idea to specify in the readme that a pip needs to be accessible, either globally or through virtualenv-awareness in emacs?

**Comment by jkitchin** [comment]
I would say it is implicit that pip be available on the PYTHONPATH that emacs knows about. what if we catch this error, and just return an empty list instead if it comes up?

**Comment by JosefMikkelsen** [comment]
Yeah I hesitated a bit about reporting this because everyone has pip after all.

The reason I noticed this error is because of a (possibly highly specific) intersection of circumstances. They are:
1. I don't have pip installed globally; I use pip exclusively inside the confines of virtualenvs
2. I wasn't making emacs virtualenv aware (didn't really see a need)

So pydoc.el wasn't finding a pip it could use. :)

To be honest I am thinking the code doesn't need changing; maybe a note in the README to ensure it can find a pip?
